app/Nova/Resources/Auth0Client.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Nova\Resources;
+
+use App\Auth0\Auth0Tenant;
+use App\Auth0\Models\Auth0ClientModel;
+use App\Domain\Contacts\Models\ContactModel;
+use App\Nova\Resource;
+use Illuminate\Database\Eloquent\Builder;
+use Laravel\Nova\Fields\BelongsTo;
+use Laravel\Nova\Fields\Field;
+use Laravel\Nova\Fields\ID;
+use Laravel\Nova\Fields\Select;
+use Laravel\Nova\Fields\Text;
+use Laravel\Nova\Http\Requests\NovaRequest;
+
+/**
+ * @mixin ContactModel
+ */
+final class Auth0Client extends Resource
+{
+    public static string $model = Auth0ClientModel::class;
+
+    public static $title = 'auth0_tenant';
+
+
+    /**
+     * @var array<string>
+     */
+    public static $search = [
+        'integration_id',
+        'auth0_client_id',
+        'auth0_client_secret',
+        'auth0_tenant',
+    ];
+
+    public static function indexQuery(NovaRequest $request, $query): Builder
```

Comment:
I'm not sure this is needed here. It was added to the `Contact` resource to make sure they are sorted on integration.